src/generator-attributes.cs

```diff
@@ -45,6 +45,27 @@ public ForcedTypeAttribute (bool owns = false)
 	public bool Owns;
 }
 
+//
+// BindAsAttribute
+//
+// // TODO Alex: finish documenting this
```

Comment:
Just to clarify that comments are useful but adding documentation is also required in `docs/website/binding_objc_libs.md` and `docs/website/binding_types_reference_guide.md` :)